test/helpers/objectCreationMethods.js

```diff
@@ -1,11 +1,25 @@
 const User = require('../../models/user');
 
+const randomDigits = () => {
+  const pad = '0000';
+  const num = Math.floor(Math.random() * 9999);
+  return (pad + num).slice(-4);
+};
+
+const randomYear = () => {
+  const min = 1900;
+  const max = new Date().getFullYear();
+  return Math.floor(Math.random() * (max - min)) + min;
+};
+
 exports.createUser = async overrides => {
+  const randomId = randomDigits();
```

Comment:
Maybe rename to `randomNumber` since this doesn't have anything to do with the `user.id`? It wasn't self-evident to me for a few seconds.

Reply:
👍